src/carma-streets/plot_message_frequencies.py

```diff
@@ -96,18 +99,21 @@ def get_simulation_time(message_wall_time : list, time_sync_wall_time: list, tim
                 message_simulation_time.append(time_sync_simulation_time[idx])
                 break
     return message_simulation_time
-def add_message_frequency_columns( messages: pd.DataFrame) -> pd.DataFrame:
+def add_message_frequency_columns( messages: pd.DataFrame, window: int = 25) -> pd.DataFrame:
```

Comment:
Default windows size picked seems to be 25, not 30. Let's either fix the comment or the default value to match.

Reply:
Yeah good catch